From 7fb9450e3d1aacde6a32cb74e9f7a0cbfdd57831 Mon Sep 17 00:00:00 2001
From: Vincent McIntyre <vincent.mcintyre@csiro.au>
Date: Mon, 11 Mar 2019 15:02:54 +1100
Subject: [PATCH] Update EFI example

Use a common architecture (amd64) in preference to an unsupported one.
Also, give an example of a path to a specific recipe within partman-auto.
---
 doc/devel/partman-auto-recipe.txt | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/doc/devel/partman-auto-recipe.txt b/doc/devel/partman-auto-recipe.txt
index bc8b541c5..5bf64ba5f 100644
--- a/doc/devel/partman-auto-recipe.txt
+++ b/doc/devel/partman-auto-recipe.txt
@@ -259,15 +259,16 @@ formated in revision 0 ext2.
 	filesystem{ ext2r0 }
 	mountpoint{ /boot } .
 
-And finally, an example of how to set up the efi boot partition needed on
-ia64.
+Finally, an example of how to set up an efi boot partition on amd64.
 
-100 100 150 fat16
-        $primary{ }
+538 538 1075 free
+        $iflabel{ gpt }
+        $reusemethod{ }
         method{ efi }
         format{ } .
 
 For other examples, see the architecture-specific recipes in partman-auto.
+The EFI example above was taken from partman-auto/recipe-amd64-efi/atomic.
 
 
 5. LIMITATIONS
-- 
2.11.0

